src/CommonITILValidation.php

```diff
@@ -1554,6 +1810,7 @@ public static function dropdownValidator(array $options = [])
         $params = [
             'prefix'            => null,
             'id'                => 0,
+            'parent'            => null,
```

Comment:
Passing the parent object is a problem, as it result in passing many information in AJAX requests. Only the `parent_id` should be passed (itemtype can be guessed using `static::$itemtype`).

![image](https://user-images.githubusercontent.com/33253653/191490532-8dfc31d1-d604-4de4-8a4a-f76f96e57bd4.png)

Reply:
fixed in: c8d2c8e92e3015bfd973fff8b70b012e9f62889b